internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -196,7 +214,10 @@ func parseUseStatement(statement string) (string, string, error) {
 	statement = removeStatementTerminator(statement)
 	words := strings.Fields(statement)
 	if len(words) < 2 {
-		return "", "", &types.StatementError{Msg: "Error: Missing database/catalog name: Usage examples: USE DB1 OR USE CATALOG METADATA."}
+		return "", "", &types.StatementError{
+			Msg:   "Missing database/catalog name",
+			Usage: []string{"USE CATALOG my_catalog or USE my_database"},
```

Comment:
Is "or" a SQL keyword, or should this be two separate usage strings?

Reply:
Yeah it should be two separate strings, missed that when refactoring!